setup.py

```diff
@@ -85,6 +98,7 @@ def run(self):
         'entrypoints',
         'tornado>=4.1',
     ],
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, <4',
```

Comment:
I think we can be optimistic and remove '<4' here. Putting this here prevents people from even attempting to install the package with a hypothetical Python 4 without us making a new release.